experimental/lib/Transforms/LSQSizing/LSQSizingSupport.cpp

```diff
@@ -573,10 +562,10 @@ int CFDFCGraph::getEarliestStartTime(mlir::Operation *op) {
 }
 
 bool CFDFCGraph::isConnectedToLSQ(mlir::Operation *op) {
-  for (mlir::Operation *destOp : op->getUsers()) {
-    if (isa<handshake::LSQOp>(destOp)) {
+  if (auto loadOp = dyn_cast<handshake::LoadOp>(op)) {
+    auto memOp = findMemInterface(loadOp.getAddressResult());
+    if (isa_and_present<handshake::LSQOp>(memOp))
```

Comment:
What about stores?

Reply:
Thanks for the catch! But I hope that's the last one :D